pkg/monitor/sqsevent/sqs-monitor.go

```diff
@@ -71,9 +71,9 @@ func (s skip) Unwrap() error {
 	return s.err
 }
 
-// Kind denotes the kind of event that is processed
+// Kind denotes the kind of monitor
 func (m SQSMonitor) Kind() string {
```

Comment:
I'm opening to renaming this method to `MonitorKind()` so that it's more clear internally that we are referring to the monitor type, not the event type. Similar change on the IMDS-based monitors.

Reply:
Yep, sounds good. Will do.